sdk/webpubsub/Azure.Messaging.WebPubSub/src/properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 using System.Runtime.CompilerServices;
+using Azure.Core;
+
+[assembly: CodeGenSuppressType("WebPubSubServiceClientOptions")]
```

Comment:
Nit - consider moving this annotation into the ClientOptions file so anyone looking at it will understand what's going on?